.github/workflows/ci.yml

```diff
@@ -113,7 +114,15 @@ jobs:
           PHP_CS_FIXER_TEST_USE_LEGACY_TOKENIZER: ${{ matrix.PHP_CS_FIXER_TEST_USE_LEGACY_TOKENIZER }}
           SYMFONY_DEPRECATIONS_HELPER: ${{ matrix.SYMFONY_DEPRECATIONS_HELPER }}
         run: |
-          vendor/bin/phpunit
+          vendor/bin/phpunit ${{ matrix.phpunit-coverage }}
```

Comment:
I disabled the coverage driver by default to speed up each build by few minutes. you need to enable the driver for one of jobs

> Error:         No code coverage driver is available